examples/data_ops/11_choices.py

```diff
@@ -2,39 +2,44 @@
 .. currentmodule:: skrub
 
 .. _example_tuning_pipelines:
+
 Tuning DataOps
```

Comment:
I think we can be more explicit here
```suggestion
Hyperparameter tuning with DataOps
```